test/integration/variable_test.rb

```diff
@@ -79,4 +79,10 @@ def test_hash_with_default_proc
   def test_multiline_variable
     assert_equal 'worked', Template.parse("{{\ntest\n}}").render!('test' => 'worked')
   end
+
+  def test_global_options_on_variable
+    template = Template.parse(%({{test}}), global_filters: ['truncate: 5'])
```

Comment:
@dylanahsmith I understand that it is not really needed until rendering so could be applied at that time and even perhaps passed in as an option to the render method. I am wondering why you don't like using liquid syntax though. The idea is to make it as easy for the end-user as possible.
This may be due to my lack of fully understanding the codebase but why should the user need to create a Proc for each filter they want to pass in as a global filter when they already know the liquid filter syntax?